index.html

```diff
@@ -731,6 +731,17 @@ <h3 id="AB-TAG-participation">2.5 Advisory Board and Technical Architecture Grou
         are expected to participate regularly and fully. Advisory Board and TAG participants <em class="rfc2119">should</em>
         attend <a href="#ACMeetings">Advisory Committee meetings</a>.</p>
 
+      <p>Individuals elected to the Advisory Board act in their personal capacity on the Board, in spite
+	      of being Member representatives, and in spite of any parallel involvement in other W3C activities as Member
```

Comment:
I like the direction of this change. I would prefer wording like "rather than representing the member who nimonated them", and being more explicit about acting in the interests of the Consortium/Web...